ci/azure-pipelines.yml

```diff
@@ -3,6 +3,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 name: $(SourceBranchName)-$(Date:yyyyMMdd).$(Rev:r)
+trigger:
+  - master
```

Comment:
Should also add pr master

Reply:
I am adding unit-tests, I will add this too.